reana_workflow_controller/tasks.py

```diff
@@ -30,7 +30,8 @@
 
 
 celery = Celery('tasks',
-                broker='amqp://test:1234@message-broker//')
+                broker='amqp://test:1234@'
+                       'message-broker.default.svc.cluster.local//')
```

Comment:
I guess we should amend this also in `reana-resources-k8s` in `deployments/reana-system/flower.yaml`?